### PR TITLE
[C-2760, C-2779] Fix header for playlist and artists tabs in explore screen

### PR DESCRIPTION
**Pull request by Kyle-Shanks**
### Description
Updated the component structure slightly to fix the headers
Screenshot 2023-06-12 at 11 20 15 PM

### Dragons

N/A

### How Has This Been Tested?

Manually tested

### How will this change be monitored?

N/A

### Feature Flags ###

N/A

### CODE REVIEW

**Review by dylanjeffers — CHANGES_REQUESTED** [review]
Hmm i think we should avoid nesting flat lists in scrollviews, does ListHeaderComponent not do what we expect?

**Comment by Kyle-Shanks** [comment]
> Hmm i think we should avoid nesting flat lists in scrollviews, does ListHeaderComponent not do what we expect?

It does, but getting the styles to look nice across all of our card lists is a bit of a pain. I'll fix it though 👍

File: packages/mobile/src/components/core/CardList.tsx [hunk 1]
```diff
@@ -35,9 +35,11 @@ const DefaultLoadingCard = () => null
 
 const useStyles = makeStyles(({ spacing }) => ({
   cardList: {
-    padding: spacing(3),
     paddingRight: 0
   },
+  contentWrapper: {
```
Comment (dylanjeffers) [inline packages/mobile/src/components/core/CardList.tsx:40]: Can this be columnWrapper?

File: packages/mobile/src/screens/explore-screen/tabs/PlaylistsTab.tsx [hunk 2]
```diff
@@ -32,12 +33,14 @@ export const PlaylistsTab = () => {
...
   return (
-    <CollectionList
-      isLoading={
-        exploreStatus === Status.LOADING || playlistsStatus !== Status.SUCCESS
-      }
-      ListHeaderComponent={<TabInfo header={messages.infoHeader} />}
-      collection={playlists}
-    />
+    <ScrollView>
+      <TabInfo header={messages.infoHeader} />
```
Comment (dylanjeffers) [inline packages/mobile/src/screens/explore-screen/tabs/PlaylistsTab.tsx:37]: I feel pretty strongly this should be in the flat list header, what's the main reason this doesn't work?

File: packages/mobile/src/screens/search-results-screen/tabs/PlaylistsTab.tsx [hunk 3]
```diff
@@ -34,6 +35,7 @@ export const PlaylistsTab = () => {
 
   return (
     <CollectionList
+      style={{ paddingTop: spacing(3) }}
```
Comment (dylanjeffers) [inline packages/mobile/src/screens/search-results-screen/tabs/PlaylistsTab.tsx:38]: I also thing it'd a bit weird we have to apply custom styles to a lot of collection lists now too. Was there something fundamentally wrong with the original styles for the list?

**Review by dylanjeffers — COMMENTED** [review]
Sorry to be nit picky on this one, just have some more qs on this one!

File: packages/mobile/src/screens/explore-screen/tabs/PlaylistsTab.tsx (hunk 2 above)
Comment (Kyle-Shanks) [reply packages/mobile/src/screens/explore-screen/tabs/PlaylistsTab.tsx:37]: oh thats weird. I updated it, but it reverted somehow. I'll fix 👍

File: packages/mobile/src/screens/search-results-screen/tabs/PlaylistsTab.tsx (hunk 3 above)
Comment (dylanjeffers) [reply packages/mobile/src/screens/search-results-screen/tabs/PlaylistsTab.tsx:38]: Word okay makes sense thanks for clarifying the long term game plan

**Pull request merged**